processor/processor_test.go

```diff
@@ -34,7 +37,7 @@ func TestProcessor(t *testing.T) {
 	}
 
 	for _, tc := range tests {
-		t.Run(tc.fileName, func(t *testing.T) {
+		t.Run(tc.testName, func(t *testing.T) {
```

Comment:
it is ok to keep filename, it is same description
```suggestion
		t.Run(tc.fileName, func(t *testing.T) {
```